pallets/grants/src/benchmarking.rs

```diff
@@ -69,7 +68,7 @@ benchmarks! {
 		let config = create_shared_config::<T>(1);
 
 		// Add some existing schedules according to b
-		for x in 0 .. MAX_SCHEDULES {
+		for x in 0 .. T::MaxSchedule::get().saturating_sub(1) {
```

Comment:
```suggestion
		for _x in 1 .. T::MaxSchedule::get() {
```

looks like x was not used in the loop